build.gradle

```diff
@@ -110,7 +103,7 @@ allprojects {
 publishing {
     publications {
             maven(MavenPublication) {
-                artifactId "corda-CSDE-kotlin-sample"
+                artifactId "corda-dev-template-kotlin-sample"

```

Comment:
I'm not sure if this is being used anywhere; publishing is disabled in the pipeline

Reply:
I can't really see why it would ever be published.